src/RemoteTech/API/API.cs

```diff
@@ -13,7 +13,8 @@ public static bool HasFlightComputer(Guid id)
             var satellite = RTCore.Instance.Satellites[id];
             if (satellite == null) return false;
             var hasFlightComputer = satellite.FlightComputer != null;
-            RTLog.Verbose("Flight: {0} HasFlightComputer: {1}", id, hasFlightComputer);
+            RTLog.Verbose("Flight: {0} HasFlightComputer: {1}", 6, id, hasFlightComputer);
```

Comment:
I think it might be easier to understand the debug level with a enumerated type rather than an int.